src/views/FlowView.jsx

```diff
@@ -119,8 +148,17 @@ class FlowView extends React.Component {
     this.stateActionCallbacks.onDeleteKey({});
   };
 
-  updateChart = (chart) => {
-    this.setState({ chart });
+  updateFlow = (flow, event) => {
+    console.log("event:", event);
+    if (syncEvents.includes(event)) return this.syncFlow(flow);
+    this.setState({ chart: flow });
```

Comment:
update flow? or syncFlow? in the store? or in the component state?
`this.syncFlow(flow)` does not return something, so why  `return this.syncFlow(flow) // always undefined`

Reply:
Return for the purpose of exiting function execution if condition is met